packages/buttons/src/styled/StyledButton.ts

```diff
@@ -49,88 +45,104 @@ export const getHeight = (props: IButtonProps & ThemeProps<DefaultTheme>) => {
  * 3. shifting :focus-visible from LVHFA order to preserve `color` on hover
  * 4. set default outline-color for smooth transition without artifacts
  */
-const colorStyles = (props: IButtonProps & ThemeProps<DefaultTheme>) => {
+const colorStyles = ({
+  theme,
+  isLink,
+  isBasic,
+  isDanger,
+  isNeutral,
+  isPrimary,
+  focusInset
+}: IButtonProps & ThemeProps<DefaultTheme>) => {
   let retVal;
-  let hue;
-
-  if (props.disabled || (props.isNeutral && props.isPrimary && !props.isDanger)) {
-    hue = 'neutralHue';
-  } else if (props.isDanger) {
-    hue = 'dangerHue';
-  } else {
-    hue = 'primaryHue';
-  }
+  const disabledBackgroundColor = getColor({ theme, variable: 'background.disabled' });
+  const disabledForegroundColor = getColor({ theme, variable: 'foreground.disabled' });
+  const offset100 = { dark: { offset: -100 }, light: { offset: +100 } };
```

Comment:
Do we need the [unary +](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus) operator?

```suggestion
  const offset100 = { dark: { offset: -100 }, light: { offset: 100 } };
```

Reply:
Agreed - shouldn't be necessary if the base operation in theming is just `+ offset` - the math works itself out 😁

Reply:
I was preferring the unary for the sake of offset clarity and scannability. But if this feels odd to the team, I'll remove.